app/views/follower/_follower.html.erb

```diff
@@ -1,10 +1,14 @@
-<div data-controller="follow">
+<div class="" data-controller="follow">
```

Comment:
Is this class supposed to be empty? 👀 If there's a specific reason for having an empty class, it's a good practice to document it in comments for clarity. If not, please remove the class of the div :D.
```suggestion
<div data-controller="follow">
```

Reply:
i forgot delete that, but now is done

Reply:
@Zama15 Cool, I am gonna approve it ☕️